config/settings.defaults.js

```diff
@@ -66,7 +66,11 @@ module.exports = {
   },
 
   enablePdfCaching: process.env.ENABLE_PDF_CACHING === 'true',
-  enablePdfCachingDark: process.env.ENABLE_PDF_CACHING_DARK === 'true'
+  enablePdfCachingDark: process.env.ENABLE_PDF_CACHING_DARK === 'true',
+  pdfCachingMinChunkSize: parseInt(
+    process.env.PDF_CACHING_MIN_CHUNK_SIZE || '1024',
+    10
+  )
```

Comment:
We've got a mixture of approaches in the config files:  `parseInt(envVar || '1234',10)` vs `parseInt(envVar,10) || 1234`.  

In this case I prefer the latter, as it prevents accidentally setting the size to 0 which could have bad consequences.   

Also, I think we should chose a safer (larger) default in the config file, in case it is turned on by accident the size of 1k is really small.

The default chunk size in pdfjs is 64k - that might be a reasonable choice - then we will only be caching larger images.
https://github.com/mozilla/pdf.js/blob/f07d50f8eeced1cd9cc967e938485893d584fc32/src/display/api.js#L61

Reply:
I've changed the fallback handling [via a force-push](https://github.com/overleaf/clsi/compare/88e4a0d49a05fdcd0ddb8cdec8090350e18dfee6..ceaf682965b8f379136d1cd9c6e42065bb40e44f).

I'm not so sure about bumping the chunk size this high. I would like to run some more tests and observe the metric for the fraction of cachable content with different values.
https://github.com/overleaf/clsi/blob/22cba9d06f16313307e66fe6175fb2c643288453/app/js/ContentCacheMetrics.js#L50-L54